selfdrive/car/hyundai/hyundaican.py

```diff
@@ -75,6 +75,8 @@ def create_lfahda_mfc(packer, enabled, hda_set_speed=0):
     "HDA_Active": 1 if hda_set_speed else 0,
     "HDA_Icon_State": 2 if hda_set_speed else 0,
     "HDA_VSetReq": hda_set_speed,
+    "LFA_USM": 2,
+    "HDA_USM": 2,
```

Comment:
Not sure which of these fix the issue, will procedurally eliminate until warning comes back

Reply:
Just as a short background...I am an IT guy, but I am new in GitHub and new in the Comma Openpilor universe.
Can you give me the correct guide, how to install this branch manually on my C3?

Reply:
SSH - then ...
cd /data && rm -rf openpilot && git clone -b master-ci https://github.com/commaai/openpilot/tree/fix-lfa-warn && sudo reboot
correct?

Reply:
Sure, just uninstall openpilot and enter `https://installer.comma.ai/commaai/fix-lfa-warn` at setup for the custom software URL.

Or you can ssh into your device, just git clone into `/data` and that way you can just use `git checkout` to switch to new branches. But for now, the installer is quicker

Reply:
To clone over ssh you want: `cd /data && rm -rf openpilot && git clone -b fix-lfa-warn -- https://github.com/commaai/openpilot && git submodule update --init && sudo reboot`

Reply:
I have a Comma3.

I installed "fix-lfa-warn". Same result. Instant FCA warning.

Here my test-drive result.
6111bd7df5043394|2022-07-21--21-29-04

Reply:
I pushed an update, let me know if that works

Reply:
Updated...testdrive...same issue ...

6111bd7df5043394|2022-07-21--22-18-10

Reply:
btw: thanks for your support ;)

Reply:
Thanks for helping test! We'll have to narrow down which message/signal is causing this, so I just made openpilot pass through the original LKAS11 message, then I'll add signals until it faults again for you. Let me know if it doesn't fault now (it might not steer)

Reply:
Like you said: No error any more. And no steering.  >>  6111bd7df5043394|2022-07-21--22-51-23

I am DerGue#4741 in discord.

Reply:
https://discord.gg/m4pMGTt9